kolibri/core/templates/kolibri/base.html

```diff
@@ -1,10 +1,8 @@
-{% load i18n kolibri_tags %}
-<html>
+{% load i18n kolibri_tags webpack_tags %}<html>
 <head>
   <title>{% block title %}{% endblock %} - {% trans "Kolibri" %}</title>
-  {% frontend_assets 'kolibri.core.KolibriCoreFrontEnd' %}
-  {% base_frontend_async %}
-  {% base_frontend_sync %}
+  {% webpack_assets 'default_frontend' %}
+  {% base_frontend_assets %}
```

Comment:
Ah yes I did, both hooks implement their own method `render_to_html`. So for now, it's all rendered through the same template tag, where as before, there were to template tags called right next to each other.